### DISCUSSION TITLE
GraphQL over http

### DISCUSSION THREAD

**Discussion by endouakulu-mirakl** [graphql-java-kickstart/graphql-java-servlet#353] [Ideas]
Few weeks ago, I read a document which specify how GraphQL should be implemented with HTTP protocol. I found this idea very interesting and I would to know if this kind of specification can be add on this library.

Thanks

**Comment by oliemansm** [comment]
@endouakulu-mirakl Don't think I quite follow what would be left to add to this library, because the whole idea of this library is to expose GraphQL using HTTP...

**Reply by endouakulu-mirakl** [reply]
The main idea of my request is to handle GraphQL errors more "HTTP compliant". This proposal should follow what I think https://github.com/graphql/graphql-over-http/blob/main/spec/GraphQLOverHTTP.md#status-codes. What do you think about this ?

**Reply by oliemansm** [reply]
Just quickly scanned it, but I believe we're already doing this.
